ReferenceDeployment/Top/ReferenceDeploymentTopology.cpp

```diff
@@ -177,16 +176,24 @@ void startSimulatedCycle(Fw::TimeInterval interval) {
     cycleLock.unLock();
 
     // Main loop
+    ReferenceDeployment::watchdogTimer.startWatchdog(interval);
```

Comment:
You should entirely remove `startSimulatedCycle` and `stop...`.  In Main.cpp, or in topology, add these calls to start rate group cycling directly.